packages/app-degree-pages/package.json

```diff
@@ -97,7 +97,7 @@
     "webpack-merge": "^5.8.0"
   },
   "volta": {
```

Comment:
Update Volta once in root package.json
All other updates can use `extends` 
"volta": {
    "extends": "../../package.json",
  }

https://docs.volta.sh/advanced/workspaces

Reply:
Ohhh, nice.